ppml/trusted-big-data-ml/python/docker-gramine/CustomImageDockerfile

```diff
@@ -0,0 +1,39 @@
+ARG BIGDL_IMAGE_NAME
```

Comment:
shall we make two folder under gramine
/docker-gramine
    /base
    /bigdl-gramine (wich is a demo purpose customer docker image?)

Reply:
Refactor to below:

`├── docker-gramine
│   ├── base
│   │   ├── Dockerfile
│   │   ......
│   ├── bigdl-gramine
│   │   └── CustomImageDockerfile
│   ├── deploy-local-spark-sgx.sh
│   ├── kubernetes
│   └── README.md`

Is it OK?